js/src/sdk/utils/error.ts

```diff
@@ -111,8 +111,14 @@ export class CEG {
                     errorKey = ERROR.BACKEND.UNKNOWN;
                     break;
             }
-            if (errorKey) {
+            if (errorKey !== ERROR.BACKEND.UNKNOWN) {
```

Comment:
The condition `errorKey !== ERROR.BACKEND.UNKNOWN` might be more clear as `errorKey && errorKey !== ERROR.BACKEND.UNKNOWN` to explicitly handle cases where errorKey might be undefined or null.